sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/text/TextSuite.scala

```diff
@@ -58,6 +58,17 @@ class TextSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("SPARK-12562 verify write.text() can handle column name beyond `value`") {
```

Comment:
why don't you just change the existing test case to rename the dataframe column and leave the following as a comment there?

SPARK-12562 verify write.text() can handle column name beyond `value`